dali/pipeline/proto/dali.proto

```diff
@@ -57,8 +58,9 @@ message OpDef {
 message PipelineDef {
   optional int64 num_threads = 1 [default = 4];
   required int32 batch_size = 2;
-  optional bool use_async = 3 [default = false];
-  optional bool pipeline = 4 [default = false];
+  // These fields were effectively unused
+  optional bool _unused_use_async = 3 [default = true];
+  optional bool _unused_pipeline = 4 [default = true];
```

Comment:
Did you happen to check, how this protobuf definition change impacts existing serialized pipelines?

Reply:
This particular change doesn't affect them, because
- we always overwrote those values. See (old) pipeline.cc:160. Only `device_id`, `num_threads`, `max_batch_size` and `seed` had magic default values that caused the serialized values to be used
- the names of fields in protobuf are not important.
I will serialize some pipelines and see how they deserialize with this PR.